app/jenkins-for-jira-ui/src/components/ConnectionPanel/ConnectionPanelContent.tsx

```diff
@@ -10,21 +10,21 @@ import {
 import { ConnectionPendingIcon } from '../icons/ConnectionPendingIcon';
 import { NoDataIcon } from '../icons/NoDataIcon';
 import { DuplicateServerIcon } from '../icons/DuplicateServerIcon';
-import { InProductHelpAction, InProductHelpActionType } from '../InProductHelpDrawer/InProductHelpAction';
+import { InProductHelpAction, InProductHelpActionButtonAppearance, InProductHelpActionType } from '../InProductHelpDrawer/InProductHelpAction';
 import { InProductHelpIds } from '../InProductHelpDrawer/InProductHelpIds';
 
 type NotConnectedStateProps = {
 	connectedState: ConnectedState;
 	contentHeader: string,
 	contentInstructionOne: string,
 	contentInstructionTwo?: string,
-	buttonAppearance: Appearance,
+	buttonOneAppearance?: Appearance,
```

Comment:
Yeah couple of existing things worth looking at

I see the _type_ thats used is called **NotConnectedStateProps** is this meant to be something more like **ConnectionPanelContent..Props**?

Yeah buttonOne, buttonTwo is tricky,  dug through and it does have quite a range of values assigned to it, hard to name it more targeted without refactoring a bit I reckon(wouldn't recommend refactoring in this PR tho :) )
Unless @yvonnevictoria has a better idea you might be stick with genericisms




Also, mix and match firstButton and buttonOne i assume they refer to the same element? can we update it to be the sames!!!

Reply:
I would suggest updating it to `primaryActionButton` and `secondaryActionButton`. This will make the button roles clearer :)

+1 on the prop types naming. I think that should be updated as it looks like a mistake.

Reply:
did 'primaryButton' and 'secondaryButton' to avoid the naming be too long.